system/Cache/Handlers/MemcachedHandler.php

```diff
@@ -115,8 +115,6 @@ public function initialize()
             } else {
                 throw new CriticalError('Cache: Not support Memcache(d) extension.');
             }
-        } catch (CriticalError $e) {
-            throw $e;
```

Comment:
This change will change the Exception message.
I don't think this catch is useless.

Reply:
Oh, I thought the last `catch (Exception $e)` catches all `CriticalError`.
You're correct. `class CriticalError extends Error`.